pages/expense.js

```diff
@@ -35,9 +39,12 @@ class ExpensePage extends React.Component {
   }
 
   render() {
-    const { data, ExpenseId, LoggedInUser } = this.props;
+    const { data, ExpenseId, LoggedInUser, createSuccess } = this.props;
+    const expenseCreated = {
+      id: ExpenseId,
+    };
 
-    if (!data.Collective) return <ErrorPage data={data} />;
+    if (!data.Collective || data.Collective.host) return <ErrorPage data={data} />;
```

Comment:
This returns an error if the collective has a host, which is a problem. Whether the collective has a host or not should not trigger any error.

Reply:
You mentioned something like that here https://github.com/opencollective/opencollective/issues/2109#issuecomment-530424085

Reply:
Yes, because we shouldn't try to show the host name if the collective has no host. But we must still display the expense info.